src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/DataGridCell.cs

```diff
@@ -1100,6 +1122,7 @@ private bool NeedsVisualTree
         private DataGridRow _owner;
         private ContainerTracking<DataGridCell> _tracker;
         private bool _syncingIsSelected;                    // Used to prevent unnecessary notifications
+        private const double _columnWidthStepSize = 10d;
```

Comment:
As the coding-style said, we should use PascalCasing to name all our constant local variables and fields.

See https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md